arch/risc-v/src/common/espressif/Kconfig

```diff
@@ -395,6 +416,104 @@ endif # ESPRESSIF_UART1
 
 endmenu # UART Configuration
 
+menu "TWAI driver options"
+	depends on ESPRESSIF_TWAI
+
+if ESPRESSIF_TWAI0
+
+config ESPRESSIF_TWAI0_TXPIN
+	int "TWAI0 TX Pin"
+	default 2
+
+config ESPRESSIF_TWAI0_RXPIN
+	int "TWAI0 RX Pin"
+	default 3
+
+choice ESPRESSIF_TWAI0_TIMING
+	prompt "TWAI0 Timing config"
+	default TWAI0_TIMING_100KBITS
+	---help---
+		These options control timing of TWAI0.
+
+config TWAI0_TIMING_100KBITS
+	bool "100 KBits"
+
+config TWAI0_TIMING_125KBITS
+	bool "125 KBits"
+
+config TWAI0_TIMING_250KBITS
+	bool "250 KBits"
+
+config TWAI0_TIMING_500KBITS
+	bool "500 KBits"
+
+config TWAI0_TIMING_800KBITS
```

Comment:
Why isn't 1MBITS supported? The most common bitrates are: 125K, 250K, 500K and 1M